### PR TITLE
added targeted emotion

### PR DESCRIPTION
**Pull request by ACLM**
fixed language URL call

### CODE REVIEW

**Comment by germanattanasio** [comment]
I think this change will break the deploy to bluemix

**Comment by nfriedly** [comment]
German is correct, [this line](https://github.com/watson-developer-cloud/alchemylanguage-nodejs/blob/master/app.js#L26) is no longer accurate with the V2 SDK. It should be replaced by something like so:

```js
// If no API Key is provided here, the SDK will check for an ALCHEMY_LANGUAGE_API_KEY 
// environment property and then fall back to the VCAP_SERVICES property provided by Bluemix.
var alchemyLanguage = watson.alchemy_language({
  //api_key: '<api-key>'
});
```

Also, this would be a good time to switch to the newer API, so

```js
// If no API Key is provided here, the SDK will check for an ALCHEMY_LANGUAGE_API_KEY 
// environment property and then fall back to the VCAP_SERVICES property provided by Bluemix.
var alchemyLanguage = new watson.AlchemyLanguageV1({
  //api_key: '<api-key>'
});
```

And, then you'd also want to update the `.env.example` file at the same time

**Comment by ACLM** [comment]
@germanattanasio can you confirm what @nfriedly said needs to be updated?

**Comment by germanattanasio** [comment]
Nathan is right and he is also the person responsible for this demo Alex.

**Comment by ACLM** [comment]
@nfriedly I believe I added the necessary changes. Please confirm. Thanks!

**Comment by nfriedly** [comment]
Not quite. This comment:

```js
// if bluemix credentials exists, then override local
```

Isn't true anymore with the v2 of the node.js SDK. It's now a fallback, not an override.  The old order of credential preference was:

1. The Bluemix-provided credentials (VCAP_SERVICES)
2. Whatever was passed into the constructor. (`process.env.API_KEY || '<api-key>'`)

The order of credential preference is now:

1. Whatever was passed into the constructor. (`process.env.API_KEY || '<api-key>'` here)
2. The SERVICE_NAME_API_KEY or SERVICE_NAME_USERNAME/PASSWORD env properties (ALCHEMY_LANGUAGE_API_KEY here) 
3. The Bluemix-provided credentials (VCAP_SERVICES)

So, with the current code:

```js
 // if bluemix credentials exists, then override local
var alchemyLanguage = new watson.AlchemyLanguageV1({
   api_key: process.env.API_KEY || '<api-key>'
 });
```

If a user were to click the "Deploy to Bluemix" button with this code and the v2 SDK specified in package.json, the credentials used would be literally `'<api-key>'` which obviously wouldn't work. The Bluemix fallback is now only invoked if no `api_key` at all is passed in. 

This was the only breaking change for v2, everything else is backwards compatible. But this part must be changed to use the v2 SDK.

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Environment variables
-API_KEY=
\ No newline at end of file
+API_KEY=
```
Comment (nfriedly) [inline .env.example]: This should say `ALCHEMY_LANGUAGE_API_KEY=`

(and you should update your private `.env` file if you have one)

File: app.js [hunk 2]
```diff
@@ -24,8 +24,8 @@ var watson = require('watson-developer-cloud');
 require('./config/express')(app);
 
 // if bluemix credentials exists, then override local
```
Comment (nfriedly) [inline app.js]: This comment is wrong now. It should say something like:

> If no API Key is provided here, the `watson-developer-cloud@2.x.x` library will check for an ALCHEMY_LANGUAGE_API_KEY environment property and then fall back to the VCAP_SERVICES property provided by Bluemix.

File: app.js [hunk 3]
```diff
@@ -24,8 +24,8 @@ var watson = require('watson-developer-cloud');
...
-var alchemyLanguage = watson.alchemy_language({
-  api_key: process.env.API_KEY || '<api-key>'
+var alchemyLanguage = new watson.AlchemyLanguageV1({
+// api_key: '<api-key>'
```
Comment (nfriedly) [inline app.js]: This looks good now.

**Comment by nfriedly** [comment]
This looks good. I need to figure out what's up with the build, but I think that's a config thing with travis. Thanks!

**Comment by nfriedly** [comment]
Op, it looks like there was actually a bug in the WDC library so it wasn't checking the correct env property :/ Fixed in v2.1.3 and then I bumped the package.json here.  Hopefully that will make the tests pass on travis.

**Pull request merged**